crates/backend/src/ast.rs

```diff
@@ -110,91 +123,144 @@ impl Hash for ImportModule {
     }
 }
 
+/// The type of item being imported
 #[cfg_attr(feature = "extra-traits", derive(Debug))]
 #[derive(Clone)]
 pub enum ImportKind {
+    /// Importing a function
     Function(ImportFunction),
+    /// Importing a static value
     Static(ImportStatic),
+    /// Importing a type/class
     Type(ImportType),
+    /// Importing a JS enum
```

Comment:
I've personally never been a huge fan of docs like on this enum, the documentation doesn't really expand any more than the english name of the type itself, e.g. that `ImportKind::Function` is "importing a function" I think is pretty obvious. I feel that docs like this tend to clutter things up without adding much value unfortuantely.